packages/contracts/test/exchange/core.ts

```diff
@@ -712,7 +684,9 @@ describe('Exchange core', () => {
                     salt: new BigNumber(3),
                 }),
             ];
-            await exchangeWrapper.batchFillOrdersNoThrowAsync(signedOrders, takerAddress);
+            await exchangeWrapper.batchFillOrdersNoThrowAsync(signedOrders, takerAddress, {
+                gas: 490000,
```

Comment:
Please assign this gas amount to a well-named constant. Also, if this is a hack, please add a `// HACK:` comment explaining why this is hard-coded.

Reply:
👍 @fabioberger I added the `// HACK:` comments here.

However, I'm not sure a constant increases code clarity in this case. These numbers were all created on a case-by-case basis for each test (by slowly increasing the gas until I found something that works). Some of the values happen to be the same now, but that's somewhat a coincidence. There was variance between each test case in terms of the actual amount of gas used and the values might change again in the future if the tests ever change.

With the comment added, I think it's pretty clear what these numbers do and why they are needed. Does it really make sense to create constants named something like `GAS_LIMIT_FOR_FILL_ORDER_NO_THROW_SHOULD_TRANSFER_THE_CORRECT_AMOUNTS`?